packages/babel-plugin-minify-simplify/src/helpers.js

```diff
@@ -2,25 +2,8 @@
 
 const VOID_0 = t => t.unaryExpression("void", t.numericLiteral(0), true);
 
-// Types as Symbols - for comparing types
-// init must be empty object -
-// computing this involves checking object.keys() to be of length 0
-// skipped otherwise
-const types = {};
-const typeSymbols = t => {
```

Comment:
I made it like this to solve a more general case of different types (Expression, LogicalExpression, etc.. - but LogicalExpression was recently removed leaving out expression as the only type used now.) in pattern matching - keeping only `Expression` seems wrong usage and this entire file `helpers` can be removed.

Reply:
Yeah i totally understand that this was for general use case. But when i was playing around, I figured out that we are not using it for anything other than expression and symbol creation was bit slow in v8. So thought of keeping it open here so we can figure out whats the best.